cmd/submit.go

```diff
@@ -125,7 +125,9 @@ func runSubmit(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 	}
 
 	exercise := workspace.NewExerciseFromDir(exerciseDir)
-
+	if _, err = exercise.MigrateLegacyMetadataFile(); err != nil {
```

Comment:
Do we want to print a notice if the legacy metadata gets moved or deleted?

Reply:
How about only printing it if in `--verbose` mode?